libkbfs/md_util.go

```diff
@@ -158,6 +158,21 @@ func getMDRange(ctx context.Context, config Config, id TlfID, bid BranchID,
 	return rmds, nil
 }
 
+// getSingleMD returns an MD that is required to exist.
+func getSingleMD(ctx context.Context, config Config, id TlfID, bid BranchID,
```

Comment:
can you make a pass over the current callers of `getMDRange` and convert to `getSingleMD` appropriately? There shouldn't be that many.